system-int-test/test/cmr/system_int_test/search/subscription/subscription_search_test.clj

```diff
@@ -4,24 +4,45 @@
    [clojure.test :refer :all]
    [cmr.common.mime-types :as mime-types]
    [cmr.common.util :refer [are3]]
-   [cmr.mock-echo.client.echo-util :as e]
-   [cmr.system-int-test.data2.core :as d]
-   [cmr.system-int-test.data2.umm-json :as data-umm-json]
-   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
-   [cmr.system-int-test.data2.umm-spec-common :as data-umm-cmn]
-   [cmr.system-int-test.data2.umm-spec-service :as data-umm-s]
-   [cmr.system-int-test.system :as s]
-   [cmr.system-int-test.utils.association-util :as au]
+   [cmr.system-int-test.data2.core :as data2-core]
    [cmr.system-int-test.utils.index-util :as index]
    [cmr.system-int-test.utils.ingest-util :as ingest]
    [cmr.system-int-test.utils.search-util :as search]
-   [cmr.system-int-test.utils.subscription-util :as subscriptions]
-   [cmr.umm-spec.versioning :as umm-version]))
+   [cmr.system-int-test.utils.subscription-util :as subscriptions]))
 
 (use-fixtures :each
               (join-fixtures
-               [(ingest/reset-fixture {"provguid1" "PROV1" "provguid2" "PROV2"})
-                (subscriptions/grant-all-subscription-fixture {"provguid1" "PROV1" "provguid2" "PROV2"})]))
+               [(ingest/reset-fixture {"provguid1" "PROV1" "provguid2" "PROV2" "provguid3" "PROV3"})
```

Comment:
We don't have any tests covering the GUEST vs registered user or group specific searches.

Reply:
Done.